client/src/components/ReviewsList.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState, useEffect } from "react"
+import ReviewTile from "./ReviewTile"
+
+const ReviewsList = props => {
+  const [reviews, setReviews] = useState([])
+
+  const getReviews = async () => {
```

Comment:
when we fetch for the board game in the show page component we should also be fetching the associated reviews, rather than issuing an additional fetch request. This means you will need a relatedQuery in the show endpoint in the router, then pass down the necessary data as props to this component